website/cue/reference/remap/functions/del.cue

```diff
@@ -15,6 +15,13 @@ remap: functions: del: {
 			required:    true
 			type: ["path"]
 		},
+		{
+			name:        "compact"
+			description: "If compact is true, after deletion, if an empty object or array is left behind, it should be removed as well, cascading up to the root."
```

Comment:
Does this only affect the parent of the field being deleted? Or does it compact the whole object? We may want to make it clearer here either way.

Reply:
It only compacts objects/arrays if they were made empty because of the deletion. I'll update the description a bit to clarify.

Reply:
👍 cool, I think that is the better behavior for `del`.